src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,55 @@
+package duke;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Duke duke;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/shef kirbo.png"));
+    private Image dukeImage = new Image(this.getClass().getResourceAsStream("/images/kirbo.png"));
```

Comment:
I like how you put in effort to personalise your bot! Looks cute!

Reply:
I like the customization using kirbo as well, it is very unique and cute!